models/admins.js

```diff
@@ -11,6 +11,7 @@ const AdminSchema = new Schema(
       type: String,
       lowercase: true,
       required: true,
+      unique: true,
```

Comment:
This is not needed. We already have a unique index on the email and organization Id. Within the same organization the same email cannot be registered twice. But the same email can be registered in a different organization. Otherwise you prevent users from registering in different organizations

Reply:
I was able to create two admins with one email

Reply:
I was able to create two admins with one email...unless that had already been resolved. I had made that commit since yesterday